README.md

```diff
@@ -34,3 +34,5 @@ broken-sinatra
   * [hint 2](https://help.github.com/articles/adding-a-remote)
 
 #### Answers can be found in the `answers` branch
+
+[Click here](https://github.com/HackerYou/broken-sinatra/compare/master...answers) to see the changes made in steps 1-5
```

Comment:
This URL might need to be updated in a later commit once this PR has been merged so we can use GitHub's UI to compare `master` against the PR merge commit's specific commit hash instead of just comparing against the entirety of the `answers` branch.  For example, https://github.com/HackerYou/broken-sinatra/compare/master...9bfe6 compares `master` against the commit specified by hash `#9bfe6`.